docs/conceptual/performance-model.rst

```diff
@@ -24,15 +24,27 @@ to use ROCm Compute Profiler to optimize your code.
 
 .. note::
 
-   In this chapter, **MI2XX** refers to any of the CDNA2 architecture-based AMD
-   Instinct MI250X, MI250, and MI210 accelerators interchangeably in cases
-   where the exact product at hand is not relevant.
+   In this chapter, **MI300** refers to any of the CDNA3 architecture-based
+   Instinct MI300X and MI300A accelerators interchangeably in cases where the
+   exact product at hand is not relevant.
+
+   Likewise, **MI200** refers to any of the CDNA2 architecture-based AMD
+   Instinct MI250X, MI250, and MI210 accelerators interchangeably.
 
    For a comparison of AMD Instinct accelerator specifications, refer to
-   :doc:`Hardware specifications <rocm:reference/gpu-arch-specs>`. For product
-   details, see the :prod-page:`MI250X <mi200/mi250x>`,
-   :prod-page:`MI250 <mi200/mi250>`, and :prod-page:`MI210 <mi200/mi210>`
-   product pages.
+   :doc:`Hardware specifications <rocm:reference/gpu-arch-specs>`.
+
+Supported features
+==================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
```

Comment:
Table seems weird with just one entry right now, but I'm sure we had ideas on how to fill it :P

Reply:
Oh, right, we probably want to (eventually) take a second pass and go through to find places where we distinguish values based on the architecture, like the waveslots discussion below (or AGPRs), and add them here.

That can probably wait till this is ~ finalized though